src/lib.rs

```diff
@@ -903,6 +1056,19 @@ fn handle_packet<H, W>(data: &[u8],
             // Unknown v commands are required to give an empty
             // response.
             Command::UnknownVCommand => Response::Empty,
+
+            // An empty response for these commands indicates they are
+            // unimplemented.
+            Command::InsertSoftwareBreakpoint(_, _, _, _) |
+            Command::InsertHardwareBreakpoint(_, _, _, _) |
+            Command::InsertWriteWatchpoint(_, _) |
+            Command::InsertReadWatchpoint(_, _) |
+            Command::InsertAccessWatchpoint(_, _) |
+            Command::RemoveSoftwareBreakpoint(_, _) |
+            Command::RemoveHardwareBreakpoint(_, _) |
+            Command::RemoveWriteWatchpoint(_, _) |
+            Command::RemoveReadWatchpoint(_, _) |
+            Command::RemoveAccessWatchpoint(_, _) => Response::Empty,
```

Comment:
I think all new command implementation should also come with a new `Handler` method, I suppose so we don't have to do as much work after the fact to wire up the parsers to the handlers.

Reply:
This is a good point.  I will do that.